libretro-common/file/retro_dirent.c

```diff
@@ -108,9 +108,13 @@ struct RDIR *retro_opendir(const char *name)
 #endif
    struct RDIR *rdir  = (struct RDIR*)calloc(1, sizeof(*rdir));
 
-   if (!rdir)
+   if (!rdir||!name)
       return NULL;
 
+   /* Handle empty string as current dir */
+   if (*name==0)
+      name=".";
```

Comment:
If you're going to do this, doesn't it ned to be null terminated?

Reply:
Yeah the double quotes should create two bytes of static data in the EXE containing a full-stop and a terminating zero, and the assignment will then point to it.